app/src/main/java/com/tranhoabinh/framgia/moviedbkotlin/ui/screen/listfavorite/ListFavoriteFragment.kt

```diff
@@ -37,15 +37,15 @@ class ListFavoriteFragment : BaseListFragment<FragmentListItemBinding, ListFavor
                 false -> initLoad()
             }
             val listMovieAdapter = ListMovieAdapter(this@ListFavoriteFragment)
-            val lineaLayoutManager = LinearLayoutManager(context)
-            val endlessScrollListener: EndlessScrollListener = object : EndlessScrollListener(lineaLayoutManager) {
+            val gridLayoutManager = GridLayoutManager(context, 2)
+            val endlessScrollListener: EndlessScrollListener = object : EndlessScrollListener(gridLayoutManager, viewModel.isLoading.value!!) {
```

Comment:
Không nên dùng toán tử !! vì nếu dùng nó nullable của kotlin không còn mang nhiều ý nghĩa nữa